syntax = "proto3";

package task_manager;

message TaskId
{
    int32 id = 1;
}